src/app/app-shell/home/home.component.tns.scss

```diff
@@ -0,0 +1,48 @@
+$prototype-color: rgb(0, 198, 224);
+$prototype-color-bg: rgb(232, 246, 255) ;
```

Comment:
`$prototype-color-bg` => `$prototype-bg-color` (it's easier if all color vars end with `-color`)

Reply:
We need to talk colors in general, I guess we will have a common set of colors? I stole these from Web-Ninja Tommyboy.